internal/raft/message/convert.go

```diff
@@ -0,0 +1,971 @@
+package message
+
+import (
+	"github.com/tomarrell/lbadd/internal/compiler/command"
+)
+
+// ConvertCommandToMessage converts a command.Command to a message.Command
+func ConvertCommandToMessage(cmd command.Command) Message {
+	switch c := cmd.(type) {
+	case *command.Scan:
+		return ConvertCommandToMessageScan(c)
+	case *command.Select:
+		return ConvertCommandToMessageSelect(c)
+	case *command.Project:
+		return ConvertCommandToMessageProject(c)
+	case *command.Delete:
+		return ConvertCommandToMessageDelete(c)
+	case *command.DropIndex:
+		return ConvertCommandToMessageDrop(c)
+	case *command.DropTable:
+		return ConvertCommandToMessageDrop(c)
+	case *command.DropTrigger:
+		return ConvertCommandToMessageDrop(c)
+	case *command.DropView:
+		return ConvertCommandToMessageDrop(c)
+	case *command.Update:
+		return ConvertCommandToMessageUpdate(c)
+	case *command.Join:
+		return ConvertCommandToMessageJoin(c)
+	case *command.Limit:
+		return ConvertCommandToMessageLimit(c)
+	case *command.Insert:
+		return ConvertCommandToMessageInsert(c)
+	}
+	return nil
```

Comment:
I imagine this hard to debug, you should probably return an error if no known command is passed in.

Reply:
Sure I didnt know the protocol for these. How should it be?

Reply:
return an error instead of nil